costmap_2d/src/costmap_2d_ros.cpp

```diff
@@ -135,7 +137,9 @@ Costmap2DROS::Costmap2DROS(std::string name, tf::TransformListener& tf) :
     {
       std::string pname = static_cast<std::string>(my_list[i]["name"]);
       std::string type = static_cast<std::string>(my_list[i]["type"]);
-      ROS_INFO("Using plugin \"%s\"", pname.c_str());
+      ROS_INFO("%s: Using plugin \"%s\"", name_.c_str(), pname.c_str());
```

Comment:
Would making this `ROS_INFO_NAMED` work here?

Reply:
Yes, good one

Reply:
Perhaps use ROS_..._NAMED("name.c_str(),..") throughout the whole file?